src/object.d

```diff
@@ -1571,6 +1603,10 @@ const:
         return null;
     }
 
+    /****************
+     * Returns:
+     *  array of TypeInfo_Class references for classes defined in this module
```

Comment:
These are sentence fragments, not complete sentences. They are clear and appropriate for this use as is.